habitat-lab/habitat/config/habitat/simulator/agents/depth_agent.yaml

```diff
@@ -0,0 +1,5 @@
+# @package habitat.simulator.agents.depth_agent
```

Comment:
TODO later (not in this PR) 
Make a 256 version of these to reduce complexity of benchmark configs